src/core/service/notifications/base.go

```diff
@@ -1,7 +1,6 @@
 package notifications
 
 import "github.com/goharbor/harbor/src/core/api"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380